### PR TITLE
slight fixes

### PR DESCRIPTION
**Pull request by shybzzz**

### CODE REVIEW

File: init-mqtt.h [hunk 1]
```diff
@@ -129,15 +129,13 @@ void initMqtt() {
...
-
+  
   DynamicJsonBuffer jsonBuffer;
   JsonObject& json = jsonBuffer.createObject();
 
   json[PAYLOAD_DEVICE] = qrConfig.DEVICE_ID;
   json[PAYLOAD_TOPIC] = topic;
-  json[PAYLOAD_VALUE] = v;
+  json[PAYLOAD_VALUE] = d;
   char payload[100];
```
Comment (ihor96berizka) [inline init-mqtt.h]: What about removing this magic number?

File: init-mqtt.h [hunk 2]
```diff
@@ -129,15 +129,13 @@ void initMqtt() {
...
   json.printTo(payload, sizeof(payload));
```
Comment (ihor96berizka) [inline init-mqtt.h]: as payload is char[100] - sizeof will always return 100.
Maybe it is better idea to use strlen() function for strings.

**Review by ihor96berizka — COMMENTED** [review]
fix comments

File: init-mqtt.h (hunk 2 above)
Comment (shybzzz) [reply init-mqtt.h]: Agree
Will look there

**Pull request merged**